system-int-test/test/cmr/system_int_test/search/tool/collection_tool_search_test.clj

```diff
@@ -90,7 +90,69 @@
         [coll1 coll2] "tool1" {:ignore-case true}
 
         "ignore-case false"
-        [] "tool1" {:ignore-case false}))
+        [] "tool1" {:ignore-case false})
+
+      ;; update tool1's Name. Make sure the collection is reindexed
+      ;; so that it can be searched using the new tool Name.
+      (tool-util/ingest-tool-with-attrs
```

Comment:
Similar comment as I made in `collection_service_search_test.clj`. I don't think this test is in the right place or necessary.

Reply:
Removed the test changes.